test/docker_single_test.go

```diff
@@ -214,7 +214,7 @@ func TestDockerSingleAutoRocksdb(t *testing.T) {
 			--starter.address=$IP \
 			--starter.mode=single \
 			--server.storage-engine=rocksdb \
-			--docker.image=arangodb/arangodb-preview:3.2.devel
+			--docker.image=arangodb/arangodb:3.2.6
```

Comment:
This image name does not seem to be right. Currently, 3.2.12 is the latest published, or 3.3.5. The preview image of the 3.2 branch is called `arangodb/arangodb-preview:3.2`, maybe we want to use that.